controllers/front/ProductController.php

```diff
@@ -576,6 +580,60 @@ protected function getMinimalProductOrDeclinationQuantity($product)
         return $minimalProductQuantity;
     }
 
+    /**
+     * Displays information, if the customer has this product in cart already.
+     */
+    protected function addCartQuantityNotification()
+    {
+        if ((bool) Configuration::get('PS_DISPLAY_AMOUNT_IN_CART') !== true) {
```

Comment:
Is it really necessary to configure this? 

Why not simply show this all the time and remove this new configuration?

If it were a more critical change I'd advise adding a configuration for backward compatibility and all, but in this case it seems overkill

Reply:
Because not all people may want to enable this behavior, and given it's in the core, it's not something where we can say “modify your theme”. For example, some people may not like that it doesn't refresh automatically. 

The change is safe, even if there is no config in database, it will work properly. 👍

Reply:
Ok but then I think more and more that this PR is a new feature and it probably should target develop branch then, sorry 😅

Reply:
I think we said we can add new features into patches if its safe no? 👀

Reply:
and absolutely necessary :)